src/software/world/game_state.cpp

```diff
@@ -105,152 +105,152 @@ bool GameState::isTheirFreeKick() const
 
 bool GameState::isTheirBallPlacement() const
 {
-    return isBallPlacement() && !our_restart;
+    return isBallPlacement() && !our_restart_;
 }
 
 // Robots must be in position for a restart
 bool GameState::isSetupRestart() const
 {
-    return state == SETUP || state == READY;
+    return state_ == SETUP || state_ == READY;
 }
 
 bool GameState::isSetupState() const
 {
-    return state == SETUP;
+    return state_ == SETUP;
 }
 
 bool GameState::isReadyState() const
 {
-    return state == READY;
+    return state_ == READY;
 }
 
 // One of our robots can kick the ball
 bool GameState::canKick() const
 {
-    return state == PLAYING || (our_restart && state == READY);
+    return state_ == PLAYING || (our_restart_ && state_ == READY);
 }
 
 bool GameState::stayAwayFromBall() const
 {
-    return state != PLAYING && !our_restart;
+    return state_ != PLAYING && !our_restart_;
 }
 
 // Our robots must stay on our half of the field
 bool GameState::stayOnSide() const
 {
-    return isSetupRestart() && restart_reason == KICKOFF && !our_restart;
+    return isSetupRestart() && restart_reason_ == KICKOFF && !our_restart_;
 }
 
 // Our robots (except the penalty kicker) must stay a set distance behind the penalty line
 bool GameState::stayBehindPenaltyLine() const
 {
-    return restart_reason == PENALTY;
+    return restart_reason_ == PENALTY;
 }
 
 void GameState::setBallPlacementPoint(Point placementPoint)
 {
-    ball_placement_point = placementPoint;
+    ball_placement_point_ = placementPoint;
```

Comment:
Can we change all camelCase variable names to snake_case?
```suggestion
    ball_placement_point_ = placement_point;
```